src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -132,26 +149,20 @@ public void startup() {
                 try {
                     List<String> caPemList = (List<String>) newv.toPOJO();
                     if (Utils.isEmpty(caPemList)) {
-                        logger.debug("CA list null or empty");
```

Comment:
Removed it because similar to https://github.com/aws-greengrass/aws-greengrass-mqtt-bridge/pull/56#discussion_r825118308, this logs doesn't provide much information.  Happy to add it back in if you think otherwise, or maybe it'd be better to log the number of certs if list is not empty.

Reply:
Ya. As long as we can make out of this case from the logs.